.murdock

```diff
@@ -154,7 +156,25 @@ export ENABLE_TEST_CACHE=${ENABLE_TEST_CACHE:-1}
 export MURDOCK_REDIS_HOST=${MURDOCK_REDIS_HOST:-127.0.0.1}
 
 NIGHTLY=${NIGHTLY:-0}
-FULL_BUILD=${FULL_BUILD:-${NIGHTLY}}
+
+# quickbuild logic
+# a "quickbuild" is only building a representative subset of all build
+# configurations.
+if [ -z "${QUICK_BUILD}" ]; then
+    export QUICK_BUILD=0
```

Comment:
hm, before there was a "export QUICKBUILD=0" in all three if cases. I moved that out, but then shellcheck reminded me that bash can't have empty if blocks, so I added the "true" lines.
Not sure what's more readable.

Reply:
resolving for being a cosmetic detail :)